luminoso_api/upload.py

```diff
@@ -88,36 +86,36 @@ def main():
     parser.add_argument('filename')
     parser.add_argument('account')
     parser.add_argument('project_name')
-    parser.add_argument('--append',
+    parser.add_argument(
+        '--append',
```

Comment:
...if we're reindenting this we should also move the trailing parenthesis to its own line.